include/enrico/nek_driver.h

```diff
@@ -25,13 +25,17 @@ class NekDriver : public Driver {
   //! NekDriver.
   //!
   //! \param comm  The MPI communicator used to initialze Nek5000
-  explicit NekDriver(MPI_Comm comm, pugi::xml_node xml_root);
+  explicit NekDriver(MPI_Comm comm, double pressure, pugi::xml_node xml_root);
```

Comment:
This makes sense.  We'll need to slightly change the specs of `enrico.xml`.  Currently, it looks like:

``` XML
<?xml version="1.0"?>
<stream>
  <driver_transport>openmc</driver_transport>
  <driver_heatfluids>nek5000</driver_heatfluids>
  <nek5000>
    <casename>rodcht</casename>
  </nek5000>
  <communication>overlapping</communication>
  <power>820.0</power>
  <pressure>12.7553</pressure>
  <max_timesteps>4</max_timesteps>
  <max_picard_iter>2</max_picard_iter>
  <openmc_procs_per_node>1</openmc_procs_per_node>
  <epsilon>0.5</epsilon>
</stream>
```

If we read pressure in `NekDriver`, we'll have to move `<pressure>` from the `<stream>` node to the `<nek5000>` node.  This is because the `<nek5000>` node is passed to the `NekDriver` constructor.

Reply:
Oh good point. I think we should move it to the `nek5000` node, since it's the only solver that needs it.